service/dps/storm/topologies/media-service/src/main/java/eu/europeana/cloud/service/dps/storm/topologies/media/service/ResourceProcessingBolt.java

```diff
@@ -85,12 +90,17 @@ public void prepare() {
                 initAmazonClient();
             }
             createMediaExtractor();
+            initGson();
```

Comment:
We can't initialize a field in a constructor of a bolt or as inline field initialization unless it is serializable, Otherwise, un exception will be raised while deploying the topology